grails-app/init/org/pih/warehouse/BootStrap.groovy

```diff
@@ -293,6 +293,7 @@ class BootStrap {
                 "inventoryItem.id"      : picklistItem.inventoryItem?.id,
                 "product.name"          : picklistItem?.inventoryItem?.product?.name,
                 "product.displayName"   : picklistItem?.inventoryItem?.product?.displayName,
+                "product.color"         : picklistItem?.inventoryItem?.product?.color,
```

Comment:
I'm cool with this one for now. But eventually, I will replace all of these fake nested objects with a proper JSON object graph. I'm going to be working on these issues over the next few months so let's keep that in mind and let me know if there's anything in particular we'll need to deal with on the client side when that happens.